ivy/functional/backends/numpy/statistical.py

```diff
@@ -253,3 +253,23 @@ def einsum(
 
 
 einsum.support_native_out = True
+
+
+def nanmedian(a, /, *, axis=None, keepdims=False, out=None, overwrite_input=False):
+    is_nan = ivy.isnan(a)
+    axis = tuple(axis) if isinstance(axis, list) else axis
+
+    if not ivy.any(is_nan):
+        ret = ivy.median(
+            a, axis=axis, keepdims=keepdims, out=out, overwrite_input=overwrite_input
+        )
+    else:
+        a = [i for i in a if ivy.isnan(i) is False]
+        ret = ivy.median(
+            a, axis=axis, keepdims=keepdims, out=out, overwrite_input=overwrite_input
+        )
+    return ret
+
+
+nanmedian.support_native_out = True
+
```

Comment:
you only need one empty line at the end of files

Reply:
done

Reply:
?? why do you still have two empty lines here?